packages/SwingSet/misc-tools/all-delivery-time-computrons.py

```diff
@@ -0,0 +1,34 @@
+#
+# Given a slogfile on stdin and a vatID, this emits a CSV of every
+# delivery (including BOYD and GC actions), with their wallclock time
+# and computrons (if any). This can be turned into a scatter chart
+# which might show trends like organic GC taking longer over time. It
+# ignores replays.
+
+import sys, json
+from collections import defaultdict
+from itertools import count
+
+cranks = [] # (deliveryNum, wallclock, computrons)
+deliveryNum = None
+start = None
+summary = None
+
+vatID = sys.argv[1]
+
+print("crankNum,deliveryNum,elapsed,computrons")
+for line in sys.stdin:
+    d = json.loads(line)
+    time = d["time"]
```

Comment:
`monotime` should be better than `time` for all duration calculations.
```suggestion
    time = d["monotime"]
```

Reply:
true, but not if the input is a set of concatenated slogfiles, each from a different process :)